kotlinx-coroutines-core/common/src/Delay.kt

```diff
@@ -133,7 +133,7 @@ public suspend fun delay(timeMillis: Long) {
 }
 
 /**
- * Delays coroutine for a given [duration] without blocking a thread and resumes it after the specified time.
+ * Delays coroutine for at least the given [duration] without blocking a thread and resumes it after the specified time.
```

Comment:
I felt that this was an important distinction which felt natural to make here. Please let me know if I should revert.